modules/nw-controlling-dns-pod-placement.adoc

```diff
@@ -40,8 +53,23 @@ $ oc edit dns.operator/default
      tolerations:
      - effect: NoExecute
        key: "dns-only"
-       operators: Equal
+       operator: Equal
        value: abc
-       tolerationSeconds: 3600 <1>
+       tolerationSeconds: 3600
 ----
-<1> If the taint is `dns-only`, it can be tolerated indefinitely. You can omit `tolerationSeconds`.
++
+This toleration matches the taint set on the nodes. If the taint is `dns-only`, it can be tolerated indefinitely. You can omit `tolerationSeconds`.
```

Comment:
This isn't necessary, but I guess I'm just trying to think of a way that we can use call outs. It's just something to think about. 

Might be worth considering the following. This way we can provide callouts for dns-only and  keep

```suggestion
. Specify a taint key and a toleration for the taint. The following toleration matches the taint set on the nodes.
+
[source,yaml]
----
 spec:
   nodePlacement:
     tolerations:
     - effect: NoExecute
       key: "dns-only" <1>
       operator: Equal
       value: abc
       tolerationSeconds: 3600 <2>
----
<1> If set to `dns-only`, it can be tolerated indefinitely.
<2> Optional. <Description> 
```